deployments/helm/nsm/charts/prefix-service/templates/deployment.tpl

```diff
@@ -0,0 +1,19 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ .Chart.Name }}
+{{/*  namespace: {{ .Release.Namespace }}*/}}
```

Comment:
`/* ... */` means comment?

Reply:
@alex-yust  Please check and respond here.

Reply:
- [x] remove commented spec